src/test/java/nextstep/qna/domain/AnswerTest.java

```diff
@@ -1,8 +1,31 @@
 package nextstep.qna.domain;
 
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import nextstep.qna.CannotDeleteException;
 import nextstep.users.domain.NsUserTest;
+import org.junit.jupiter.api.Test;
 
 public class AnswerTest {
-    public static final Answer A1 = new Answer(NsUserTest.JAVAJIGI, QuestionTest.Q1, "Answers Contents1");
-    public static final Answer A2 = new Answer(NsUserTest.SANJIGI, QuestionTest.Q1, "Answers Contents2");
+
+    public static final Answer A1 = new Answer(NsUserTest.JAVAJIGI, QuestionTest.Q1,
+        "Answers Contents1");
+    public static final Answer A2 = new Answer(NsUserTest.SANJIGI, QuestionTest.Q1,
+        "Answers Contents2");
+
+    @Test
+    public void delete_다른_사람이_쓴_글() throws Exception {
```

Comment:
명확한 테스트의 의도를 드러내 주시면 좋을 것 같아요
```suggestion
    public void delete_다른_사람이_쓴_답변은_삭제할_수_없다() throws Exception {
```